SmartTutor/ContentModel/EducationContent.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SmartTutor.ContentModel;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+
+namespace SmartTutor.ContentModel
+{
+    public class EducationContent
+    {
+        public int EducationQuality { get; set; }
```

Comment:
A better name for these two fields would be ContentQuality and ContentDifficulty.